modules/core/02-client/keeper/events.go

```diff
@@ -68,6 +70,17 @@ func EmitUpdateClientProposalEvent(ctx sdk.Context, clientID string, clientState
 	)
 }
 
+// EmitUpgradeClientProposalEvent emits an upgrade client proposal event
+func EmitUpgradeClientProposalEvent(ctx sdk.Context, title string, height int64) {
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeUpgradeClientProposal,
+			sdk.NewAttribute(types.AttributeKeyUpgradePlanTitle, title),
+			sdk.NewAttribute(types.AttributeKeyUpgradePlanHeight, fmt.Sprint(height)),
```

Comment:
nit: sprintf formatting

```suggestion
			sdk.NewAttribute(types.AttributeKeyUpgradePlanHeight, fmt.Sprintf("%d", height)),
```